cobigen-templates/templates-devon4j/src/main/templates/crud_ngrx_client_app/templates/src/app/layout/header/header.component.ts.ftl

```diff
@@ -0,0 +1,64 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Router } from '@angular/router';
+import { Store } from '@ngrx/store';
+import { TranslateService } from '@ngx-translate/core';
+import { logOutAction } from '../../auth/store/actions';
+import { AuthService } from '../../core/security/auth.service';
+import { AppState } from '../../home/sampledata/store/reducers/index';
```

Comment:
Is this `sampledata` correct? Or actually it should be `${variables.component}` or something like that?